site/app/templates/admin/LateDays.twig

```diff
@@ -77,4 +77,37 @@
     $("#user_id").autocomplete({
         source: {{ student_full|raw }}
     });
+    flatpickr("#datestamp", {
+            plugins: [ShortcutButtonsPlugin(
+                    {
+                        button: [
+                            {
+                                label: "Now"
+                            }, 
+                            {
+                                label: "End of time"
+                            }
+                        ],
+                        label: "or",
+                        onClick: (index, fp) => {
+                            let date;
+                            switch (index) {
+                                case 0: 
+                                    date = new Date();
+                                    break;
+                                case 1: 
+                                    date = new Date("01/01/98");
+                                    break;
+                            }
+                            fp.setDate(date, true);
+                        }
+                    }
+                )],
+            allowInput: true,
+            enableTime: true,
```

Comment:
Let's not `enableTime` or `enableSeconds`. They are not used in the timestamp on this page.

Reply:
ok, cool..